[v3] Fix for CORS exceeding payload

Tolfx opened the pull request:
# Description
Using POST for runtime and body to make the limit of query params higher. Has breaking changes that needs testing if implemented.

Fixes #4428

## Type of change
  
Please select the option that is relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] This change requires a documentation update

# How Has This Been Tested?
  
This has been tested by using https://github.com/xob0t/wails3_cors_error then changed the following files
**go.mod**
```
replace github.com/wailsapp/wails/v3 => /home/tolfx/dev/wailsv3/v3
```

**package.json**
```json
"@wailsio/runtime": "file:/wailsv3/v3/internal/runtime/desktop/@wailsio/runtime"
```

- [X] Windows
- [ ] macOS
- [X] Linux

NAME="Linux Mint"
VERSION="22 (Wilma)"
ID=linuxmint
ID_LIKE="ubuntu debian"
PRETTY_NAME="Linux Mint 22"
VERSION_ID="22"
trixie/sid
Linux TOLFX 6.8.0-71-generic #71-Ubuntu SMP PREEMPT_DYNAMIC Tue Jul 22 16:52:38 UTC 2025 x86_64 x86_64 x86_64 GNU/Linux
  
# Checklist:

- [ ] I have updated `website/src/pages/changelog.mdx` with details of this PR
- [X] My code follows the general coding style of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

leaanthony commented:
@Tolfx - could you please check the latest alpha release as it also includes switching to POST. I would be keen to know if it addresses all your concerns :+1:

Tolfx commented:
> @Tolfx - could you please check the latest alpha release as it also includes switching to POST. I would be keen to know if it addresses all your concerns 👍

I've looked into the alpha and tested and does indeed solve the problem. Will close this PR, but will create an issue if any problems occurs within the development.
